src/language/utils.ts

```diff
@@ -77,7 +77,7 @@ export function scanAllTokens(scanner: ts.Scanner, callback: (s: ts.Scanner) =>
 /**
  * @returns true if any modifier kinds passed along exist in the given modifiers array
  */
-export function hasModifier(modifiers: ts.ModifiersArray, ...modifierKinds: ts.SyntaxKind[]) {
+export function hasModifier(modifiers: ts.ModifiersArray|null|undefined, ...modifierKinds: ts.SyntaxKind[]) {
```

Comment:
why would it be `null`? it could only be `undefined`, right? with this change, the condition on the next line can also be changed to be more strict, i.e. `modifiers === undefined || modifierKinds === undefined`